receiver/vmmetricsreceiver/transform_stats_to_metrics.go

```diff
@@ -0,0 +1,276 @@
+// Copyright 2018, OpenCensus Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package vmmetricsreceiver
+
+// TODO(songy23): remove this file and use Metrics Gauge and Cumulative APIs.
```

Comment:
This file is a copy of https://github.com/census-ecosystem/opencensus-go-exporter-ocagent/blob/master/transform_stats_to_metrics.go. We will remove this once we migrate to use the new Cumulative and Gauge APIs in Go.